.github/workflows/push.yml

```diff
@@ -90,6 +89,20 @@ jobs:
           # Exit with status code 1 if there are differences (i.e. unformatted files)
           git diff --exit-code
 
+  golangci:
+    name: lint
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-go@v5
+        with:
+          go-version: stable
```

Comment:
Curious: do you know why `setup-go` is needed?

If it is, let's use the same Go version we use in the other jobs `1.23.2`. Then it won't drift.

Reply:
Great question. This comes from the [set up instruction](https://github.com/marketplace/actions/golangci-lint)s. It seems that some setup from setup-go action is reused in this action:

> By default, the [problem matcher of Go (actions/setup-go)](https://github.com/actions/setup-go/blob/main/matchers.json) already handles the golangci-lint output (colored-line-number).

I've tried removing it https://github.com/databricks/cli/pull/1953/commits/0027c654f1013684da405e47750568252b4c04cb and it passes without complains https://github.com/databricks/cli/actions/runs/12160990672/job/33914524909

Another reason could be custom plugins: https://golangci-lint.run/plugins/module-plugins/

Reply:
I restored setup-go with explicit version  82ac762aa8f7a5db